include/openmc/dagmc.h

```diff
@@ -142,6 +156,7 @@ class DAGUniverse : public Universe {
                             //!< universe in OpenMC's surface vector
 
   // Accessors
+  std::shared_ptr<moab::DagMC> dagmc_ptr() const { return dagmc_instance_; }
```

Comment:
I think since this method is primarily used internally, a bare pointer (or reference) would make more sense (and be more efficient).

Reply:
You can follow on with a mini-PR once that makes it into DAGMC... although is that going to break compatibility with previous versions of DAGMC then?

Reply:
Mmm, yeah you're right. Probably not worth doing that.